chaos_genius/alerts/email_templates/digest_template.html

```diff
@@ -53,14 +53,16 @@ <h3>Top 10 Anomalies</h3>
         <span style="color: lightgrey;">
             changed to <strong>{{point.y}}</strong>
         {% if point.percent_change is string and "inf" not in point.percent_change %}
-        (<span>{{ point["change_message"] }}</span>)
+            <span>
         {% elif "+inf" in str(point.percent_change) or point.percent_change > 0 %}
-        (<span style="color: lightgreen;">{{ point.change_message }}</span>)
+            <span style="color: lightgreen;">
         {% elif "-inf" in str(point.percent_change) or point.percent_change < 0 %}
-        (<span style="color: red;">{{ point.change_message }}</span>)
+            <span style="color: red;">
```

Comment:
can refactor this same way we did for [`email_alert.html`](https://github.com/chaos-genius/chaos_genius/pull/862/files#diff-af0f76080279d03cf20d59d6442c348f60c38cf5581b1f84a4765a9766169886)

Reply:
i thought i had. good catch.

Reply:
hahaa ,,, i guess lots of staged changes in the Git of brain .. and error prone tracking of which are committed